fdm-app/app/components/custom/field/schema.tsx

```diff
@@ -0,0 +1,22 @@
+import { z } from "zod"
+
+const FormSchema = z.object({
+    b_name: z
+        .string({
+            required_error: "Naam van perceel is verplicht",
+        })
+        .min(3, {
+            message: "Naam van perceel moet minimaal 3 karakters bevatten",
+        }),
+    b_lu_catalogue: z.string({
+        required_error: "Hoofdgewas is verplicht",
+    }),
+    b_id_source: z.string({
+        required_error: "ID van bron is verplicht",
+    }).optional(),
+    b_geometry: z.string({
```

Comment:
what does "optional" implies here since the input is mandatory?

Reply:
Because the user form does not include the geometry, so it can't be validated client side. However, the geometry is added to the formData at submission, thus it is server side validated